src/MaterialUI.re

```diff
@@ -1309,6 +1311,42 @@ module Select = {
     );
 };
 
+module Slide = {
+  [@bs.module "material-ui/transitions/Slide"] external slide : ReasonReact.reactClass = "default";
+  module Direction = {
+    type t =
+      | Up
+      | Down
+      | Left
+      | Right;
+    let to_string =
+      fun
+      | Up => "up"
+      | Down => "down"
+      | Left => "left"
+      | Right => "right";
+  };
+  let make =
+      (
+        ~in_: option(bool)=?,
+        ~direction_: option(Direction.t)=?,
+        ~timeout: option({. "enter": float, "exit": float})=?,
```

Comment:
The docs specify you should be able to provide a single float (or number in js) value as well.